core/js/src/files/PathsJs.kt

```diff
@@ -123,11 +133,24 @@ internal class FileSource(private val path: Path) : RawSource {
 
     override fun close() {
         closed = true
+        fs.closeSync(fd)
```

Comment:
Does calling `closeSync` won't again result in an error?

Reply:
It would be an error, thanks to pointing to it.